gen/test-html.hs

```diff
@@ -21,23 +22,68 @@ data Presentation = Presentation
     { title :: String
     , author :: String
     , tags :: [Tag]
-    , slides :: Maybe String
-    , audio :: Maybe String
-    , player :: Maybe String
+    , slides :: Maybe FilePath
```

Comment:
How about declaring "type URL = String" for this purpose? Would you accept that?